web/beacon-app/src/features/auth/components/Register/SuccessfullAccountCreation.tsx

```diff
@@ -33,22 +33,20 @@ function SuccessfullAccountCreation() {
           <p>
             {' '}
             <Trans>
-              Didn't receive an email?{' '}
-              <span className="font-bold text-[#1F4CED]">Resend the verification email</span> or
-              email{' '}
+              Didn't receive an email? Check your spam folder or email{' '}
```

Comment:
I updated the copy to remove `resend verification email` for now. Once the feature is add, this will be updated.